assets/app/styles/_create.less

```diff
@@ -0,0 +1,225 @@
+//
+// Create App
```

Comment:
@sg00dwin You pulled this back into core.less for now, right? If so, we should remove this file from the commit.